OpenRA.Game/Scripting/ScriptTypes.cs

```diff
@@ -162,7 +163,17 @@ public static LuaValue ToLuaValue(this object obj, ScriptContext context)
 				return (bool)obj;
 
 			if (obj is string)
-				return (string)obj;
+			{
+				// HACK: Workaround as Lua 5.1 cuts of strings with Umlauts.
```

Comment:
See also #17484. It is possible that crash happens because of this same bug cutting the string in the middle of a multi-byte sequence. If my suggestion here works (compile your own patched dll and replace the one installed via nuget) we should fix this in our Eluant fork, push a new version to nuget, and then have a separate PR here to update the nuget reference and close that issue.

Reply:
See https://github.com/OpenRA/Eluant/pull/4. Removed the workaround from here.